PingPayment.PaymentLinksApi/PingPayments.PaymentLinksApi.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
Jag undrar om denna ska bort... hm Varför har vi den nu?

Reply:
ImplicitUsings används för att kunna använda global usings. se (Första stycket) https://www.hanselman.com/blog/implicit-usings-in-net-6